v4-client-js/src/clients/modules/post.ts

```diff
@@ -113,14 +115,22 @@ export class Post {
     memo?: string,
     account?: () => Promise<Account>,
   ): Promise<StdFee> {
-    const msgsPromise = messaging();
-    const accountPromise = account ? await account() : this.account(wallet.address!);
-    const msgsAndAccount = await Promise.all([msgsPromise, accountPromise]);
-    const msgs = msgsAndAccount[0];
+    let msgs: EncodeObject[];
+    let sequence = Date.now();
```

Comment:
Nit: could we document that protocol expects timestamp nonce in UTC milliseconds, which is the unit returned by `Date.now()`